controllers/categoryController.js

```diff
@@ -0,0 +1,49 @@
+const categoryModel = require("../models/categoryModel");
+
+//get built-in categories in JSON format
+async function getBuiltInCategory(req, res) {
+    var result = await categoryModel.getBuiltInCategory();
+    if(!result)
+        res.json({"result": "fail"});
+    else {
+        var categories = JSON.parse(result);
+        res.json({"result": "success", categories});
```

Comment:
categories로 묶어주는 걸로 했던 것 같아서요!
```suggestion
        res.json({"result": "success", "categories" : categories});
```

Reply:
네 확인하고 수정했습니다 감사합니다